.github/conda_skip.yml

```diff
@@ -0,0 +1,74 @@
+exclude:
```

Comment:
Do we know if this file is actually being used?

Reply:
Did some research this is still being used, they are pushing a fix to use conda_skip. You can follow it here: https://github.com/nf-core/modules/pull/7175#issuecomment-2578336422. This is a critical update as we need it to fix our tests, should we merge this test framework and update it again later when the conda fix is added or just hotfix in the container fix?

Reply:
Let's merge what we have now since we need singularity testing working sooner than later. For now, we'll skip all conda testing and make a PR when `conda_skip` is working again.